nservicebus/operations/tuning.md

```diff
@@ -12,37 +12,30 @@ tags:
  - performance
  - throttling
  - concurrency
-reviewed: 2016-11-15
+reviewed: 2018-10-10
 ---
 
-NServiceBus uses defaults that ensure good performance in most common cases. While this is usually the preferred mode of operation there are situations where tuning needs to be applied.
-
+NServiceBus uses defaults that ensure good performance in common cases. While this is usually the preferred mode of operation there are situations where tuning might be desired.
```

Comment:
>NServiceBus uses defaults

Think it should be "use defaults"

Reply:
Isn't it uses defaults? Unsure though

Reply:
Had a faint memory that "only use plural s on the last word" but I can't find any refs on that

https://english.stackexchange.com/questions/53124/multiple-plural-any-rules

only made me more confused :)

Reply:
It should be "uses defaults", since "NServiceBus" is a singular entity.

It would have to be plural for "use", e.g. "NServiceBus packages use defaults...".